pyproject.toml

```diff
@@ -36,47 +37,44 @@ exclude = ["tests", "tests.*"]
 hathor-cli = 'hathor.cli.main:main'
 
 [tool.poetry.dev-dependencies]
-flake8 = "~4.0.1"
+flake8 = "~5.0.4"
```

Comment:
Have you checked the changes in each of these packages? I feel a shallow check would be enough.

Reply:
We moved this conversation to https://github.com/HathorNetwork/internal-issues/issues/140.